lib/beaker/dsl/install_utils/aio_defaults.rb

```diff
@@ -17,13 +17,13 @@ module AIODefaults
           },
           'windows' => { #windows
             'puppetbindir'      => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/bin',
-            'privatebindir'     => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
+            'privatebindir'     => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/puppet/bin',
```

Comment:
Yep, that's why I listed this PR as "don't merge". I have put up a card for the vanagon/windows sync up to talk about what to do about the issue.